grails-app/views/customer/show.gsp

```diff
@@ -1,22 +1,23 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="pt-br">
     <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>Informações do customer</title>
     </head>
     <body>
-        <form>
+        <form method="POST" action="${createLink(controller: "customer", action: "sendUpdate")}">
+            <input type="hidden" name="id" value="${customer.id}">
             <div>
                 <label>Nome</label>
                 <br>
-                <input type="text" name="name" value="${customer.name}" readonly>
+                <input type="text" class="update-input" name="name" value="${customer.name}" readonly>
```

Comment:
Por acessibilidade, acho que seria interessante adicionar id nos inputs e referenciar esses ids no for das labels. Isso ajuda os screen readers a ler os campos do formulário pra pessoas com limitações visuais.

Reply:
Como é um form de teste, e a gente vai usar o atlas form depois, vou adicionar na aba de tarefas da criação do form no front.